lib/ohai/system.rb

```diff
@@ -87,19 +85,6 @@ def load_plugins
     end
 
     def run_plugins(safe = false, attribute_filter = nil)
```

Comment:
this whole function can get cleaned up a lot.

Reply:
There's several methods where that's true